### ISSUE TITLE
Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.

### ISSUE THREAD

**Issue by kayzersoze26** [tensorflow/tensorflow#21223]
I get the following error

An error ocurred while starting the kernel
2018󈚫󈛁 18:04:30.223185: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018󈚫󈛁 18:04:30.275857: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:897] successful NUMA node read from SysFS had negative value (𔂫), but there must be at least one NUMA node, so returning NUMA node zero
2018󈚫󈛁 18:04:30.276812: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1392] Found device 0 with properties: 
name: GeForce GT 750M major: 3 minor: 0 memoryClockRate(GHz): 0.967
pciBusID: 0000:01:00.0
totalMemory: 1.95GiB freeMemory: 1.57GiB
2018󈚫󈛁 18:04:30.306371: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:897] successful NUMA node read from SysFS had negative value (𔂫), but there must be at least one NUMA node, so returning NUMA node zero
2018󈚫󈛁 18:04:30.307350: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1392] Found device 1 with properties: 
name: GeForce GT 750M major: 3 minor: 0 memoryClockRate(GHz): 0.967
pciBusID: 0000:07:00.0
totalMemory: 1.95GiB freeMemory: 1.93GiB
2018󈚫󈛁 18:04:30.307702: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1471] Adding visible gpu devices: 0, 1
2018󈚫󈛁 18:04:30.751898: I tensorflow/core/common_runtime/gpu/gpu_device.cc:952] Device interconnect StreamExecutor with strength 1 edge matrix:
2018󈚫󈛁 18:04:30.751936: I tensorflow/core/common_runtime/gpu/gpu_device.cc:958] 0 1 
2018󈚫󈛁 18:04:30.751943: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971] 0: N Y 
2018󈚫󈛁 18:04:30.751947: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971] 1: Y N 
2018󈚫󈛁 18:04:30.752166: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1084] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1346 MB memory) ‑> physical GPU (device: 0, name: GeForce GT 750M, pci bus id: 0000:01:00.0, compute capability: 3.0)
2018󈚫󈛁 18:04:30.767241: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1084] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 1709 MB memory) ‑> physical GPU (device: 1, name: GeForce GT 750M, pci bus id: 0000:07:00.0, compute capability: 3.0)
2018󈚫󈛁 18:15:26.140459: E tensorflow/stream_executor/cuda/cuda_driver.cc:1078] failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018󈚫󈛁 18:15:26.140506: E tensorflow/stream_executor/cuda/cuda_timer.cc:55] Internal: error destroying CUDA event in context 0x55b5818bb6d0: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018󈚫󈛁 18:15:26.140514: E tensorflow/stream_executor/cuda/cuda_timer.cc:60] Internal: error destroying CUDA event in context 0x55b5818bb6d0: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018󈚫󈛁 18:15:26.140532: F tensorflow/stream_executor/cuda/cuda_dnn.cc:189] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.

my system information
Nvidi-smi RESULTS

 NVIDIA-SMI 390.77                 Driver Version: 390.77                    |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GT 750M     Off  | 00000000:01:00.0 N/A |                  N/A |
| N/A   50C    P8    N/A /  N/A |    432MiB /  1999MiB |     N/A      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce GT 750M     Off  | 00000000:07:00.0 N/A |                  N/A |
| N/A   38C    P8    N/A /  N/A |      2MiB /  1999MiB |     N/A      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0                    Not Supported                                       |
|    1                    Not Supported                                       |

CUDA Results
CUDA Version 9.2.148

CUDNN version Results

#define CUDNN_MAJOR 7
#define CUDNN_MINOR 1
#define CUDNN_PATCHLEVEL 4
--
#define CUDNN_VERSION    (CUDNN_MAJOR * 1000 + CUDNN_MINOR * 100 + CUDNN_PATCHLEVEL)

#include "driver_types.h"
what could be the error here

thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 16 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by skye** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!

**Comment by JialieY** [comment]
I met the same issue, can you please provide a solution?

**Comment by JackDemo** [comment]
So am i

**Comment by glard** [comment]
So am I

**Comment by Benehiko** [comment]
Also, have the same issue running Ubuntu18.04, 3GB GTX 780 with Cuda 10.0.130, Tensorflow 1.12.0 compiled from source for GPU

**Comment by issa-s-ayoub** [comment]
Now I am facing the same problem with tensorflow-gpu 1.12 (Build from binaries) on windows 10 and cuda v9.0, GPU: GTX 1080 Ti.

**Comment by wumeng2** [comment]
@issa-s-ayoub  hello, do you solve it? I also have the same problem. Thank you!

**Comment by Oldhuntor** [comment]
I have the same problem too，anybody solved it？

**Comment by JackDemo** [comment]
It's caused by insufficient memory. By recalculating the neural network layer and allocating memory space, my problem can be solved.

**Comment by henbucuoshanghai** [comment]
so am I
cuda10,cudnn7.3.1  tf10  
how to solve it?

**Comment by JackDemo** [comment]
检查你的程序  计算一下程序中内存占用情况  很大程度上是因为内存溢出导致启动失败



发自我的小米手机
在 henbucuoshanghai <notifications@github.com>，2019年6月3日 下午6:23写道：

so am I
cuda10,cudnn7.3.1 tf10
how to solve it?

**Comment by henbucuoshanghai** [comment]
是吗  2080ti  gpu内存是11g够的吧    那怎样在代码里设置呢？

**Comment by henbucuoshanghai** [comment]
the same code.   it can run successfluy sometimes,but sometimes can not with the error  in issue.
i am confused.......        why.....

**Comment by JackDemo** [comment]
我能帮助你的 只有利用我掌握的少量知识 内存泄漏一般主要出现在自己编写的问题中  可以通过计算判断大概需要多少内存
[cid:image001.png@01D51A63.930524D0]

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

________________________________
发件人: henbucuoshanghai <notifications@github.com>
发送时间: Monday, June 3, 2019 11:01:04 PM
收件人: tensorflow/tensorflow
抄送: Jack; Comment
主题: Re: [tensorflow/tensorflow] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream. (#21223)


the same code. it can run successfluy sometimes,but sometimes can not with the error in issue.
i am confused....... why.....

―
